example/src/main/java/org/apache/rocketmq/example/simple/AsyncProducer.java

```diff
@@ -55,6 +61,7 @@ public void onException(Throwable e) {
                 e.printStackTrace();
             }
         }
+        countDownLatch.await(5, TimeUnit.SECONDS);
```

Comment:
LGTM.
btw, I am wondering if we can improve internally in product for the AsyncProducer.java, that will make users easier to use.